### DISCUSSION TITLE
Shorten onboarding PR description?

### DISCUSSION THREAD

**Discussion by cabadam** [renovatebot/renovate#24089] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.42.0 (via Docker image)

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Onboarding PR with all necessary information viewable.

### What you tried so far.

We have a large monorepo that is trying to add Renovate to it.  It successfully creates the onboarding PR.  It starts listing out the various "Detected Package File", but then cuts off in the middle of one of them and has no text following it.  There are MANY of them (though I think it managed to list most before cutting off, based on the alphabetized order).

I copy/pasted the PR description it created and notepad++ indicates its length is 30,292 characters.  Not a nice round 32K number, but close.  Maybe missing some formatting?

Is there a way to customize the content of this PR so the repo admin can view what packages/PRs would get created before merging the onboarding PR?  I looked around at the configuration options and saw one that allows customization of the PR title, but not the description/body.

Thanks

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
the truncation is because the 32k limit for PR description on bitbucket server

**Comment by cabadam** [comment]
Right, I figured the limit was about there, which is really the crux of the question.  Given the limit, how can I make sure projects get the most use out of those 32K characters?  Listing every project/package file isn't particularly useful if they are trying to figure out their renovate.json and make sure it is right.  For example, I'd really rather suppress that, leaving just the list of PRs that would be created.

**Reply by cabadam** [reply]
I wound up back here when we ran into this again on another project being onboarded and they were not able to view the instructions of the onboarding PR. Has any consideration been made to changing the PR description to ensure the most important information is included?
